portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/PageProfileVisitorBehavior.testcase

```diff
@@ -0,0 +1,68 @@
+@component-name = "portal-analytics-cloud"
+definition {
+
+	property analytics.cloud.enabled = "true";
+	property minimum.slave.ram = "32";
+	property portal.release = "false";
+	property portal.upstream = "true";
+	property testray.main.component.name = "Analytics Cloud";
+
+	setUp {
+		ACUtils.launchAC();
+
+		ACSettings.takeTokenFromAnalyticsCloud();
+
+		TestCase.setUpPortalInstance();
+
+		ACUtils.launchDXP();
+	}
+
+	tearDown {
+		var testPortalInstance = PropsUtil.get("test.portal.instance");
+
+		ACUtils.tearDownDXP();
+
+		if ("${testPortalInstance}" == "true") {
+			PortalInstances.tearDownCP();
+		}
+
+		ACUtils.tearDownAC();
+	}
+
+	@description = "Feature ID: Legacy | Automation ID: LRAC-8516 | Test Summary: Assert Sites Page Overview time filter to Custom Filter"
+	@ignore = "true"
+	@priority = "5"
+	test CanTimeFilterUseCustomRangeInPageOverview {
+
+		// TODO LRAC-8516 CanTimeFilterUseCustomRangeInPageOverview
+
+	}
```

Comment:
hi @LucasAcacio , could you please remove this test stub? We removed it from the test map.

Thanks!